docs/upgrade/upgrade_zonemaster_backend_ver_8.0.0.md

```diff
@@ -0,0 +1,104 @@
+# Upgrade to 8.0.0
+
+## New dependencies
+
+Zonemaster::Backend requires new dependencies. Depending on the used OS, run
+the corresponding command.
+
+### Centos
+
+```sh
+sudo cpanm Plack::Middleware::ReverseProxy
+```
+
+### Debian / Ubuntu
+
+```sh
+sudo apt-get install libplack-middleware-reverseproxy-perl
+```
+
+### FreeBSD
+
+```sh
+su -l
+pkg install p5-Plack-Middleware-ReverseProxy
+```
```

Comment:
This gives one block of two lines that you can paste at once. The first line is executed, and if correct root password is provided, then a new root shell is started. The second line is executed when the root shell has terminated, i.e. as the user before `su -l`. 

The following would give you what you want, but I suggest to have `su -l` as an introduction instead, especially if several root commands are to be executed.
````
```sh
su -l root -c 'pkg install p5-Plack-Middleware-ReverseProxy'
```
````

Reply:
Btw, can we envisage adding `sudo` on FreeBSD?
```
pkg install sudo
```
It would make it so much easier to maintain.